stream-parent/stream-client/src/main/java/org/interledger/stream/sender/SimpleStreamSender.java

```diff
@@ -289,22 +296,20 @@ default int totalPackets() {
     private AtomicInteger numFulfilledPackets;
     private AtomicInteger numRejectedPackets;
 
+    private Optional<Denomination> receiverDenomination = Optional.empty();
```

Comment:
To keep the consistency, wouldn't it be nicer to initialize the variable to `Optional.empty()` in the constructor for `SendMoneyAggregator` instead of having it here.